src/components/StepByStepGuide.tsx

```diff
@@ -127,10 +127,25 @@ const StepByStepGuide: React.FC<StepByStepGuideProps> = ({
       icon: <GitFork className="h-8 w-8" />,
       multi: false,
       componentId: "discrete",
-      options: optionsByFacet["pathwayType"].map((o) => ({
-        ...o,
-        description: descriptions["pathwayType"][o.title] || undefined,
-      })),
+      options: optionsByFacet["pathwayType"]
+        .sort((a, b) => {
```

Comment:
@jacobvjk I think it would be useful to extract the `order` array into `src/utils/sortUtils.ts`, and basically make a "sortPathwayType` function or closure, and then import and use it both here and in the dropdown. If you want, I can wire that up, or I think you have all the right parts in here

Reply:
updated, but not sure it follows your preferred style